src/Zlib.Shared/Inflate.cs

```diff
@@ -1698,19 +1698,25 @@ internal int Inflate(FlushType flush)
 
 
 
-        internal int SetDictionary(byte[] dictionary)
+        internal int SetDictionary(byte[] dictionary, bool unconditional = false)
         {
             int index = 0;
             int length = dictionary.Length;
-            if (mode != InflateManagerMode.DICT0)
-                throw new ZlibException("Stream error.");
-
-            if (Adler.Adler32(1, dictionary, 0, dictionary.Length) != _codec._Adler32)
-            {
-                return ZlibConstants.Z_DATA_ERROR;
-            }
-
-            _codec._Adler32 = Adler.Adler32(0, null, 0, 0);
+			//MSZip requires the dictionary to be set unconditionally
```

Comment:
Could you please restore the indentation?

Reply:
Follow the previous type of indentation, don't suddenly introduce tabs. @darkms is right.